app/src/main/java/tech/relaycorp/ping/domain/model/Ping.kt

```diff
@@ -0,0 +1,10 @@
+package tech.relaycorp.ping.domain.model
+
+import java.time.ZonedDateTime
+
+data class Ping(
+    val messageId: String,
```

Comment:
I'm assuming this is the ping id, not the parcel id, so I'd probably call this `id` or `pingId` to avoid confusion.